grudge/op.py

```diff
@@ -552,10 +565,10 @@ def _apply_mass_operator(
     )
 
 
-def mass(dcoll: DiscretizationCollection, *args):
+def mass(dcoll: DiscretizationCollection, *args) -> ArrayOrContainerT:
     r"""Return the action of the DG mass matrix on a vector (or vectors)
     of :class:`~meshmode.dof_array.DOFArray`\ s, *vec*. In the case of
-    *vec* being an object array of :class:`~meshmode.dof_array.DOFArray`\ s,
+    *vec* being an :class:`~arraycontext.container.ArrayContainer`,
     the mass operator is applied in the Kronecker sense (component-wise).
```

Comment:
I'm actually not sure I buy that applying componentwise is accurately described as a Kronecker product.

Reply:
I've gone ahead and just simplified it by referring to "component-wise" application. I think it's much more clear anyways :)

In case you're wondering where I got the idea of Kronecker products, it's [here](https://www.brown.edu/research/projects/scientific-computing/sites/brown.edu.research.projects.scientific-computing/files/uploads/Review%20of%20entropy%20stable%20discontinuous%20Galerkin%20methods.pdf), page 13